src/Stack/Package.hs

```diff
@@ -27,56 +27,62 @@ module Stack.Package
+import qualified Distribution.PackageDescription        as D
 import           Distribution.PackageDescription.Parsec
-import           Distribution.Simple.Glob (matchDirFileGlob)
-import           Distribution.System (OS (..), Arch, Platform (..))
-import qualified Distribution.Text as D
-import qualified Distribution.Types.CondTree as Cabal
-import qualified Distribution.Types.ExeDependency as Cabal
+import           Distribution.Simple.Glob               (matchDirFileGlob)
+import           Distribution.System                    (Arch, OS (..),
+                                                         Platform (..))
+import qualified Distribution.Text                      as D
+import qualified Distribution.Types.CondTree            as Cabal
+import qualified Distribution.Types.ExeDependency       as Cabal
 import           Distribution.Types.ForeignLib
 import qualified Distribution.Types.LegacyExeDependency as Cabal
 import           Distribution.Types.MungedPackageName
 import qualified Distribution.Types.UnqualComponentName as Cabal
-import qualified Distribution.Verbosity as D
-import           Distribution.Version (mkVersion, orLaterVersion, anyVersion)
-import           Path as FL
+import qualified Distribution.Verbosity                 as D
+import           Distribution.Version                   (anyVersion, mkVersion,
+                                                         orLaterVersion)
+import           Path                                   as FL
 import           Path.Extra
-import           Path.IO hiding (findFiles)
+import           Path.IO                                hiding (findFiles)
+import           RIO.PrettyPrint
+import qualified RIO.PrettyPrint                        as PP (Style (Module))
+import           RIO.Process
 import           Stack.Build.Installed
 import           Stack.Constants
 import           Stack.Constants.Config
-import           Stack.Prelude hiding (Display (..))
+import qualified Stack.ModuleInterface                  as Iface
+import           Stack.Prelude                          hiding (Display (..))
 import           Stack.Types.Compiler
 import           Stack.Types.Config
 import           Stack.Types.GhcPkgId
 import           Stack.Types.NamedComponent
 import           Stack.Types.Package
 import           Stack.Types.Version
-import qualified System.Directory as D
-import           System.FilePath (replaceExtension)
-import qualified System.FilePath as FilePath
+import qualified System.Directory                       as D
+import           System.FilePath                        (replaceExtension)
+import qualified System.FilePath                        as FilePath
 import           System.IO.Error
-import           RIO.Process
-import           RIO.PrettyPrint
-import qualified RIO.PrettyPrint as PP (Style (Module))
 
-data Ctx = Ctx { ctxFile :: !(Path Abs File)
-               , ctxDistDir :: !(Path Abs Dir)
+data Ctx = Ctx { ctxFile        :: !(Path Abs File)
```

Comment:
Also not a fan of forced alignment of records, it makes diffs larger than necessary